.github/build.in.yml

```diff
@@ -413,6 +413,8 @@ jobs:
           COUNTEREXAMPLES_OUTCOME: ${{ steps.counterexamples.outcome }}
           LINT_OUTCOME: ${{ steps.lint.outcome }}
           TEST_OUTCOME: ${{ steps.test.outcome }}
+          MATHLIB_REPO: leanprover-community/mathlib4
+          NIGHTLY_TESTING_REPO: leanprover-community/mathlib4-nightly-testing
```

Comment:
These two variables (`MATHLIB_REPO` and `NIGHTLY_TESTING_REPO`) are set, but I don't see where they are used: should they be used somewhere?

Reply:
NIGHTLY_TESTING_REPO is used in lean-pr-testing-comments.sh to generate a comparison URL.

MATHLIB_REPO is unused, and I'll remove it.